ksqldb-cli/src/main/java/io/confluent/ksql/cli/Options.java

```diff
@@ -127,6 +133,12 @@
       description = "Execute commands from a file and exit.")
   private String scriptFile = null;
 
+  @SuppressWarnings("unused") // Accessed via reflection
+  @Option(
+      name = {DEFINE_OPTION, DEFINE_SHORT_OPTION},
+      description = "Define variables.")
```

Comment:
should we add a little bit more description here? Maybe something like "equivalent to the `DEFINE` CLI command"